src/test/java/io/debezium/connector/jdbc/transforms/ConvertCloudEventToSaveableFormTest.java

```diff
@@ -43,39 +60,47 @@ void testConvertNotCloudEventRecord() {
         }
     }
 
-    @Test
-    @FixFor("DBZ-7065")
-    void testConvertCloudEventRecordWithEmptyConfig() {
+    @ParameterizedTest
+    @ValueSource(strings = { "json", "avro" })
+    @FixFor({ "DBZ-7065", "DBZ-7130" })
+    void testConvertCloudEventRecordWithEmptyMapping(String serializerType) {
         try (ConvertCloudEventToSaveableForm transform = new ConvertCloudEventToSaveableForm()) {
             final Map<String, String> config = new HashMap<>();
+            config.put("serializer.type", serializerType);
             transform.configure(config);
 
             final SinkRecordFactory factory = new DebeziumSinkRecordFactory();
 
-            final SinkRecord cloudEventRecord = factory.cloudEventRecord("test.topic");
-            assertThat(cloudEventRecord.valueSchema().name()).endsWith(".CloudEvents.Envelope");
-            assertThat(cloudEventRecord.valueSchema().fields().size()).isEqualTo(7);
-            assertThat(cloudEventRecord.valueSchema().field("id").schema()).isEqualTo(Schema.STRING_SCHEMA);
+            final SinkRecord cloudEventRecord = factory.cloudEventRecord("test.topic", SerializerType.withName(serializerType));
+            if (serializerType.equals("avro")) {
```

Comment:
Should not there be asserts for JSON too? - same for other methods

Reply:
In JSON case there is no schema of the value =>

```
java.lang.NullPointerException: Cannot invoke "org.apache.kafka.connect.data.Schema.name()" because the return value of "org.apache.kafka.connect.sink.SinkRecord.valueSchema()" is null
```

Reply:
And what is the source of that difference? Is it intentional?

Reply:
Yes, it is intentional. The source is CloudEventsConverter that produces a struct with no schema if it deserialized a record from JSON. The schema exists only in Avro case

Reply:
But IIRC it is possible to serialize to JSON with schema included. Don't we need to test such case too?

Reply:
Fixed in the latest commit

Reply:
Probably one will apply SMT after CE converter which docs [says](https://github.com/debezium/debezium/blob/main/debezium-core/src/main/java/io/debezium/converters/CloudEventsConverter.java#L373) that there will be no schema

Reply:
If it is right, the latest commit should be removed